lib/executor/helpers_test.go

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2020 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package executor
+
+import "github.com/loadimpact/k6/stats"
+
+func sumMetricValues(samples chan stats.SampleContainer, metricName string) (sum float64) {
+	bufferedSmaples := stats.GetBufferedSamples(samples)
+	for _, sc := range bufferedSmaples {
```

Comment:
```suggestion
	bufferedSamples := stats.GetBufferedSamples(samples)
	for _, sc := range bufferedSamples {
```
though I would've just used `stats.GetBufferedSamples(samples)` in the `range`

Reply:
Resolved by https://github.com/loadimpact/k6/pull/1529/commits/2153af88c7414d8b47342f555b3aa886d5de0e40